### PR TITLE
Adding Visual Studio Code extension to community open-source doc

### PR DESCRIPTION
**Pull request by tsurdilo**
Signed-off-by: Tihomir Surdilovic <tsurdilo@redhat.com>

### CODE REVIEW

File: community/open-source.md [hunk 1]
```diff
@@ -43,3 +43,4 @@ CloudEvents, please include in the list below.
   dependency manager for Kubernetes which uses sensors to act on CloudEvents.
 - [cloudevent.js](https://github.com/smartiniOnGitHub/cloudevent.js): A
   CloudEvents library for Node.js.
+- [CloudEvents JSON Schema](https://marketplace.visualstudio.com/items?itemName=tsurdilovic.cloudevents-schema-vscode): Visual Studio Code extension for CloudEvents format.
```
Comment (duglin) [inline community/open-source.md]: minor thing, but can you wrap this after the colon?

Comment (tsurdilo) [reply community/open-source.md]: updated, let me know if its OK now.

Comment (duglin) [reply community/open-source.md]: thanks!

**Comment by duglin** [comment]
LGTM
I'll wait a day to make sure no one has any issues and then merge since it's not touching a spec and is non-controversial.

**Comment by duglin** [comment]
ok merging since this isn't touching a spec, just some community info - thanks!

**Pull request merged**